### ISSUE TITLE
rename junxion / osc mode

### ISSUE THREAD

**Issue by sensestage** [sensestage/ssdn_python#18]
"I don't think junXion is being supported anymore - so I would suggest renaming the junXion OSX mode to something else -- because sending the minibee id in the address is actually extremely useful in many cases. For example, it's the only way to differentiate between minibees in TouchDesigner. I would suggest giving the two radio buttons as 'OSC send format' with a text field including a preview of the format (also a nice reminder of what address to look for in your client software) ... something like this:

-- OSC Send Format --
O - minibee ID as first element:        /minibee/data id val1 val2
O - minibee ID as part of address:    /minibee/data/id val1 val2"

**Comment by sensestage** [comment]
Though there are still JunXion users, I think - so indeed that format needs to be there.

There is an issue that for sending data to the minibee in the junxion format, there are still some hardcoded limitations to how many (I haven't properly automated that part yet).

I have to see how complicated this will be to implement properly.

**Comment by sensestage** [comment]
Perhaps also datanetwork and libmapper can be removed.
Libmapper hasn't been tested or updated since a long time.
Datanetwork could be an advance option if we want to keep it around.

**Comment by jreus** [comment]
Getting rid of the datanetwork stuff would be great if nobody is using it. I feel like Pydon should be a piece of software that you run quickly and then it gets out of your way.. the more simple and self-explanatory Pydon is, with as few options as possible, the better. :-)

**Comment by sensestage** [comment]
In SuperCollider you can use argTemplate to filter for a specific minibee, e.g. for minibee 1:

```
OSCdef( \minibeedata, { |data,time,source| data.postln; }, '/minibee/data', argTemplate: [ 1 ]  );
```

**Comment by sensestage** [comment]
For the names of these options:
``` '/minibee/*' ```
and
``` '/minibee/*/<ID>' ```
does that make sense?

**Comment by jreus** [comment]
@sensestage hmm, good question... I see two user bases that might be using Pydon.
1. users who are not familiar with text-based operations like wild cards
2. users who are familiar with command-line operations and wouldn't balk at seeing an asterisk

For group 2 your proposal seems pretty clear, but it assumes the knowledge of the whole Sense/Stage OSC address space to understand why there is a wildcard there. I think group 1, and more generally novice Sense/Stage users who probably only use the /minibee/data message, would be better served by using that, along with a short description of the difference between the two formats.
 
`minibee ID as the first OSC argument. ex: /minibee/data id data1 data2 ...`
`minibee ID as part of OSC address. ex: /minibee/data/id data1 data2 ...`

When users need to use more advanced OSC commands they will probably end up looking to the documentation. At least that's my impression!

**Comment by sensestage** [comment]
fixed in 99f18abd8e50d5ef2b1de3b18b1cba25560eba2a
